docs/framework/migration-guide/how-to-determine-which-versions-are-installed.md

```diff
@@ -156,6 +156,10 @@ Users can install and run multiple versions of the .NET Framework on their compu
 
 - The following example checks the value of the `Release` keyword to determine whether .NET Framework 4.6.2 or higher is installed (returning `True` if it is and `False` otherwise). You can substitute another value listed in the table to check for a different minimum-required .NET Framework version.
 
+```PowerShell
+Get-ChildItem "hklm:SOFTWARE\Microsoft\NET Framework Setup\NDP\v4\Full\" | Get-ItemPropertyValue -Name Release | % { $_ -ge 394802 } 
+```
+
```

Comment:
weird, the table now is broken and tokens didn't resolve: 

![image](https://user-images.githubusercontent.com/12971179/28850331-d20baf60-76d0-11e7-9fa1-2e0f26607736.png)